File: CursoPOOuber/Python/ubervan.py
from car import Car

class UberVan(Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, licencia, driver, typeCarAccepted, seatsMaterial):
        super.__init__(licencia, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial

File: CursoPOOuber/Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

class UberVan extends Car {
    Map<String, Map<String, Integer>> typeCarAccepted;
    Map<String> seatsMaterial;
    private Integer pasajeros;

    public UberVan(String licencia, Account driver, Integer pasajeros){
        super(licencia, driver);
        this.pasajeros = pasajeros;
    }

    public UberVan(String licencia, Account driver, Map<String, 
    Map<String, Integer>> typeCarAccepted, 
    Map<String> seatsMaterial){
        super(licencia, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial; 
    }
    
    @Override
    public void setPasajeros(Integer pasajeros){
        if(pasajeros == 6){
            this.pasajeros = pasajeros;
        }else{
            system.out.println("Necesita tener 6 pasajeros");
        }
    }

}